src/components/skeletons/SkeletonProductTiles/SkeletonProductTiles.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import Tiles, { Tile } from 'src/components/ui/Tiles'
+
+import SkeletonProductTile from './SkeletonProductTile'
+
+interface Props {
+  loading?: boolean
+  children?: JSX.Element
+  variant?: 'vertical' | 'horizontal'
+}
+
+function SkeletonProductTiles({
+  children,
+  loading = true,
+  variant = 'vertical',
+}: Props) {
+  return loading ? (
+    <Tiles>
+      {Array.from({ length: 3 }, (_, index) => (
```

Comment:
```suggestion
      {Array.from({ length: CONSTANT }, (_, index) => (
```
Create a constant.
Also, prefer receive itensQuantity as prop and pass this constant.